### PR TITLE
docs(README): Add version compatibility matrix

### PR DESCRIPTION
**Pull request by aimeeu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,13 @@
 
 Spinnaker plugin for configuring and customizing observability.
 
+## Version Compatibility
```
Comment (fieldju) [inline README.md:10]: I am hesitant to have this in the README as I don't plan on manually keeping this up to date.

I am more interested in somehow utilizing the following

https://github.com/armory-plugins/armory-observability-plugin/blob/master/gradle.properties#L8-L16
https://github.com/armory-plugins/armory-observability-plugin-releases/blob/master/plugins.json#L10

cc: @claymccoy @link108

Comment (fieldju) [reply README.md:10]: @ncknt, did you end up putting any thought into this?

Comment (aimeeu) [reply README.md:10]: However you do it, the end user needs to be able to to easily locate plugin version compatibility with Spinnaker.   If that means adding a line to the README that points to a generated file, that's OK.
FWIW, pointing the end user to 
```
"requires": "clouddriver>=0.0.0,echo>=0.0.0,fiat>=0.0.0,front50>=0.0.0,gate>=0.0.0,igor>=0.0.0,kayenta>=0.0.0,orca>=0.0.0,rosco>=0.0.0",
```
isn't very user-friendly. How many users know which service versions comprise a Spinnaker release?  Can you add a field that explicitly states the Spinnaker version?

**Comment by fieldju** [comment]
I think we will have the matrix for now until there is something better

**Pull request merged**